proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fr_FR.properties

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Vous êtes déjà connecté à ce serveur !
+velocity.error.already-connected-proxy=Vous êtes déjà connecté à ce proxy !
+velocity.error.already-connecting=Vous êtes déjà en train d'essayer de vous connecter à un serveur !
+velocity.error.cant-connect=Impossible de se connecter à {0}: {1}
+velocity.error.connecting-server-error=Impossible de vous connecter à {0}. Veuillez réessayer plus tard.
+velocity.error.connected-server-error=La connexion à {0} a rencontré un problème.
+velocity.error.internal-server-connection-error=Une erreur de connexion interne au serveur s'est produite.
+velocity.error.logging-in-too-fast=Vous vous connectez trop rapidement, réessayez plus tard.
+velocity.error.online-mode-only=Vous n'êtes pas connecté à votre compte Minecraft. Si vous l'êtes, essayez de redémarrer votre client Minecraft..
+velocity.error.player-connection-error=Une erreur interne s'est produite lors de votre connexion.
+velocity.error.modern-forwarding-needs-new-client=Ce serveur est uniquement compatible avec Minecraft 1.13 et plus.
+velocity.error.modern-forwarding-failed=Votre serveur n'a pas envoyé de requête de forwarding au proxy. Assurez-vous que le serveur est configuré pour le forwarding Velocity.
+velocity.error.moved-to-new-server=Vous avez été explusé de {0}: {1}
+velocity.error.no-available-servers=Il n'y a pas de serveurs disponibles auxquels vous connecter. Essayez à nouveau plus tard ou contactez un administrateur.
```

Comment:
I would phrase it more like this `[...] Réessayez ultérieurement ou contactez un administrateur.`

_suggestion_